modules/cluster-limitations-local-zone.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc
+
+[id="cluster-limitations-local-zone_{context}"]
+= Cluster limitations in AWS Local Zones
+
+Some limitations exist when you attempt to deploy a cluster with a default installation configuration in Amazon Web Services (AWS) Local Zones. If you do not not configure your cluster for deployment in AWS Local Zones, edge worker nodes might not process user workloads. 
+
+The following list details limitations when deploying a cluster in AWS Local Zones:
+
+- A higher cluster-wide network maximum transmission unit (MTU) value than the MTU specified for edge worker nodes can impact how edge worker nodes in your cluster communicate. Settings a lower cluster-wide network MTU can resolve this issue. 
```

Comment:
```suggestion
- The Maximum Transmission Unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region is 1300, thus the cluster-wide network MTU is changed according to the network plugin used on the deployment.
```

Reply:
For style guide requirements, I had to slightly tweak the text.

What do you think of the following change?

> The Maximum Transmission Unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in a region is `1300`. This causes the cluster-wide network MTU to change according to the network plugin that is used on the deployment.

Reply:
OK.
fore reference to "the Region":
https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html
```text
Generally, the Maximum Transmission Unit (MTU) is as follows:
(...)
- 1300 bytes between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region.
```

Reply:
Thanks, updated.